src/hooks/usePool.ts

```
@@ -0,0 +1,115 @@
+    if (!connected || !publicKey) {
+      setBalance(0);
+      setLoading(false);
+      return;
+    }
+    try {
+      setError(null);
+      const data = await apiService.getPoolBalance({
+        userWallet: publicKey.toBase58(),
+        quoteToken,
+      });
+      const bal = parseFloat((data && data.balance) || data || 0);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)